java/src/jmri/jmrit/entryexit/configurexml/EntryExitPairsXml.java

```diff
@@ -227,6 +227,9 @@ public boolean load(Element shared, Element perNode) {
                         String id = null;
                         if (destinationList.get(j).getAttribute("uniqueid") != null) {  // NOI18N
                             id = destinationList.get(j).getAttribute("uniqueid").getValue();  // NOI18N
+                            if (id.length() > 3 && !id.startsWith("IN:")) {     // NOI18N
```

Comment:
This is fine, no need to change it, but just out of curiosity why the test for length > 3? startsWith(..) is OK with short strings.

Reply:
Sorry, that wasn't clear.  This won't prefix a string like "ab", even though it doesn't start with "IN:", because it's too short.  Is that intentional?

Reply:
I think I got my thinking twisted reading the Javadoc for startsWith.

I will remove the length check.